Makefile

```diff
@@ -1,12 +1,33 @@
 ARCH := darwin/amd64 linux/386 linux/amd64 windows/386 windows/amd64
 VERSION := $(shell git describe --always --dirty)
 LDFLAGS := -ldflags "-X main.version=${VERSION}"
+BUILD_DIR := build
+BINARY_NAME ?= yajsv
 
-yajsv: *.go
+.PHONY: build
+build: *.go
 	go build ${LDFLAGS}
 
+.PHONY: release
 release: *.go
-	gox -output 'build/{{.Dir}}.{{.OS}}.{{.Arch}}' -osarch "${ARCH}" ${LDFLAGS}
+	gox -output '${BUILD_DIR}/{{.Dir}}.{{.OS}}.{{.Arch}}' -osarch "${ARCH}" ${LDFLAGS}
 
+.PHONY: clean
 clean:
-	rm -rf build yajsv
+	$(eval BINARY := ${BINARY_NAME}$(if $(findstring windows,$(shell go env GOOS)),.exe,))
+	rm -rf ${BUILD_DIR} ${BINARY} coverage.out
```

Comment:
Can remove the `${BINARY_NAME}` variable and make this delete `yajsv` and `yajsv.exe` explicity since it doesn't matter if the files exist or not. (That's a neat `Makefile` trick though that I didn't know about and could be useful in other contexts).

Reply:
done.